Geometry/EcalMapping/src/EcalElectronicsMapping.cc

```diff
@@ -1084,7 +1086,8 @@ void EcalElectronicsMapping::GetListofFEDs(const EcalEtaPhiRegion& region, std::
 		if (minL == MIN_DCCID_EBM) minL=MAX_DCCID_EBM;
 		else minL = minL -1;
 		int maxL = (FED_LT-1)*2 + MIN_DCCID_EBM;
-		int iL = minL;
+		// FIXME: Value stored to 'iL' during its initialization is never read
+		int iL; // = minL;
```

Comment:
```iL``` doesn't appear to be used outside of the loop so could be moved inside the loop.